models/games.models.js

```diff
@@ -6,18 +6,60 @@ exports.fetchCategories = () => {
   });
 };
 
-exports.fetchReviews = () => {
-  return db
-    .query(
-      "SELECT owner, title, reviews.review_id, category, review_img_url, reviews.created_at, reviews.votes, designer, COUNT(comment_id) AS comment_count FROM reviews\
-    LEFT JOIN comments ON comments.review_id = reviews.review_id\
-    GROUP BY reviews.review_id\
-    ORDER BY reviews.created_at DESC;"
-    )
-    .then((reviews) => {
-      return reviews.rows;
-    });
-};
+exports.fetchReviews = (queryObj) => {
+    if (Object.keys(queryObj).length !== 0) {
+      const possQuery = ["order", "sort_by", "category"];
+  
+      bool = Object.keys(queryObj).every((key) => possQuery.includes(key));
```

Comment:
Be very careful declaring global variables like this! Especially now we're working in a server. If another request came in at the same time, both would be pointing to the exact same `bool` variable.
Also, try to be more descriptive regarding what this boolean actually represents - `isValidQuery` for example